### DISCUSSION TITLE
Terabox Request failed with status code 500

### DISCUSSION THREAD

**Discussion by anwen-anyi** [AlistGo/alist#3026] [Q&A] [answered]
我从添加 Terabox挂载  到访问 点击文件的 [程序日志，点击访问](https://down.onmicrosoft.cn/qiniu/u20221115caac3864_1673529770478_18391517_file.log)

不知道是什么情况，前几天还可以访问来着，是我网络被封了吗？
虽然我是自己家里的电脑搭建的但是我用了v2rayN让Alist也吃到了代理，前几天是可以访问的 没有任何问题
今天看到[issue](https://github.com/alist-org/alist/issues/3017) 有人提了，我就去试试看然后我发现我的也无法预览和下载了，点击文件就会出现日志中那一大串的报错
晚上的时候看到了您新pr的也合并了，我就拿最新合并并且编译出来的去试了下也是一样的问题，然后我就用replit(国外的IP)去搭建了一个进行了测试 倒是没有问题 可以预览下载 不会报错...

麻烦您啦大佬 看看什么情况QAQ  @Code2qing  
等待您的答复 :two_hearts: :two_hearts: :two_hearts: :two_hearts: :two_hearts:

**Comment by anwen-anyi** [comment]
刚刚不知道怎么点的 在replit也发现了上述描述的情况 [**replit log点击查看**](https://down.onmicrosoft.cn/qiniu/u20221115caac3864_1673531245451_16134891_file.log)

**Reply by Code2qing** [reply]
这种情况是没有获取到下载链接，我自己这边是从来都没复现过，试试cookie只设置ndus字段看可不可以

**Reply by anwen-anyi** [reply]
> 这种情况是没有获取到下载链接，我自己这边是从来都没复现过，试试cookie只设置ndus字段看可不可以

不行QAQ.....后面我在replit搭建测试的时候用的是和我本地的cookie一样..倒是可以预览 下载操作

**Comment by Code2qing** [comment]
你是不是保存了terabox设置之后，在浏览器上官网将terabox账号退出登录了，退出登录后会导致此前的cookie被吊销不再可用。

**Reply by anwen-anyi** [reply]
没有 后面我还故意退出 重新登录 
再获取了一次依旧是这样 
我盲猜可能是封了我IP导致的应该🤣
我是本机用V2 给git Bash代理 这样我本地就可以访问了
我在等等过阵子换个IP试试看 能不能访问  在提这个问题之前我用Replit测试了一下 没问题 所以我怀疑我被封IP了😁

**Comment by anwen-anyi** [comment, marked as answer]
好吧 大概知道是什么问题了 还是网络问题哈哈，主要是 **`terabox`** 锁IP，下面梳理一下供后面人参考，
当然了如果你搭建Alist的机器是外面的不用担心这个问题....

**总的来说是网络问题导致的**

启动客户端使用的是Git Bash 和 V2代理工具
 - ✅可以访问没问题 
   - Git Bash 输入：**`export https_proxy=http://127.0.0.1:10809 http_proxy=http://127.0.0.1:10809 all_proxy=socks5://127.0.0.1:10809`** 然后 输入启动Alist 命令
   - V2 代理工具设置：全局
![OK](http://gchat.qpic.cn/gchatpic_new/0/0-0-65788E749A3216939DB052EDB281A7E4/0)
 
- 🚫不可以访问有问题 
   - Git Bash 输入：**`export https_proxy=http://127.0.0.1:10809 http_proxy=http://127.0.0.1:10809 all_proxy=socks5://127.0.0.1:10809`** 然后 输入启动Alist 命令
   - V2 代理工具设置：绕过DL
![Error](http://gchat.qpic.cn/gchatpic_new/0/0-0-CF8B4649082DF28EBE92EE08EE5A8506/0)

**Comment by lockcp** [comment]
openclash配置文件里加一条` - DOMAIN-SUFFIX,terabox.com,Proxies`就解决了

**Reply by anwen-anyi** [reply]
QAQ 我是PC`v2rayN` ~
